app/graph/nodes/answer/smalltalk_node.py

```
@@ -52,9 +51,31 @@ def smalltalk_node(state: GraphState) -> dict:
+                persona_lines.append(f"Greeting: {greeting}")
+            if style:
+                persona_lines.append(
+                    f"[Speech style instruction — written in Korean]: {style}\n"
+                    f"→ Translate the above Korean style instruction into {lang_name} first, "
+                    f"then follow it exactly in {lang_name}. "
+                    f"If it says to add a word/phrase at the end of sentences, "
+                    f"translate that word/phrase into {lang_name} and add it.\n"
+                    f"- CRITICAL: The style MUST be visible in your response.\n"
+                    f"- CRITICAL: Do NOT use the original Korean words — always translate them into {lang_name}."
+                )
+            persona_block = "\n".join(persona_lines) if persona_lines else f"You are a cute mascot guide at a tourist site."
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)